cmd/influx/config.go

```diff
@@ -289,6 +288,27 @@ func (b *cmdConfigBuilder) cmdListRunEFn(*cobra.Command, []string) error {
 	return b.printConfigs(configPrintOpts{configs: cfgs})
 }
 
+func (b *cmdConfigBuilder) registerConfigSettingFlags(cmd *cobra.Command) {
+	cmd.Flags().BoolVarP(&b.active, "active", "a", false, "Set as active config")
+	cmd.Flags().StringVarP(&b.org, "org", "o", "", "The optional organization name")
+
+	cmd.Flags().StringVarP(&b.name, "config-name", "n", "", "The config name (required)")
+	cmd.MarkFlagRequired("config-name")
+
+	cmd.Flags().StringVarP(&b.token, "token", "t", "", "The token for host (required)")
+	cmd.MarkFlagRequired("token")
+
+	cmd.Flags().StringVarP(&b.url, "host-url", "u", "", "The host url (required)")
+	cmd.MarkFlagRequired("host-url")
+
+	// deprecated moving forward, not explicit enough based on feedback
+	// the short flags will still be respected but their long form is different.
+	cmd.Flags().StringVar(&b.name, "name", "", "The config name (required)")
+	cmd.Flags().MarkDeprecated("name", "use the --config-name flag")
+	cmd.Flags().StringVar(&b.url, "url", "", "The host url (required)")
```

Comment:
@sanderson these are the flags that are being deprecated. They still work, but won't show up in the `-h` anymore

Reply:
👍